js/constants.js

```diff
@@ -12,7 +12,13 @@ export const TODAY = new Date();
 export const COMPLETE = "complete";
 export const EDIT = "edit";
 export const DELETE_TODO = "delete";
+export const EDIT_SAVE = "editSave";
+export const EDIT_CHECK = "editCheck";
+export const EDIT_DELETE = "editDelete";
 export const CLICK_EVENT = "click";
 export const HIDE = "hide";
 export const AFTER_BEGIN = "afterbegin";
 export const MS_PER_DAY = 86400000;
+
+export const DONE_ICON = "icons/done.svg";
+export const DELETE_ICON = "icons/delete.svg";
```

Comment:
use common prefix instead of a suffix to group things together.

```suggestion
export const ICON_DONE = "icons/done.svg";
export const ICON_DELETE = "icons/delete.svg";
```